### PR TITLE
Upgrade cert manager

### PR DESCRIPTION
**Pull request by a-cool-train**
*Issue #, if available:*

*Description of changes:*
- Upgrade cert manager version to the latest version.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: projects/cert-manager/cert-manager/build/cert-manager.yaml [hunk 1]
```diff
@@ -0,0 +1,5422 @@
+# Copyright 2021 The cert-manager Authors.
```
Comment (jonahjon) [inline projects/cert-manager/cert-manager/build/cert-manager.yaml:1]: What is the point of this entire file?

Comment (a-cool-train) [reply projects/cert-manager/cert-manager/build/cert-manager.yaml]: It isn't necessary. I took it out.

**Comment by jaxesn** [comment]
Does this need updating https://github.com/aws/eks-anywhere-build-tooling/blob/main/projects/cert-manager/cert-manager/manifests/cert-manager.yaml

**Comment by a-cool-train** [comment]
> Does this need updating https://github.com/aws/eks-anywhere-build-tooling/blob/main/projects/cert-manager/cert-manager/manifests/cert-manager.yaml

Yes, I just updated it now.

**Comment by jaxesn** [comment]
> > Does this need updating https://github.com/aws/eks-anywhere-build-tooling/blob/main/projects/cert-manager/cert-manager/manifests/cert-manager.yaml
> 
> Yes, I just updated it now.

Can you add a note about it in the Updating section in the Readme?

**Comment by TerryHowe** [comment]
looks good although Jackson has a request

**Comment by a-cool-train** [comment]
> > > Does this need updating https://github.com/aws/eks-anywhere-build-tooling/blob/main/projects/cert-manager/cert-manager/manifests/cert-manager.yaml
> > 
> > 
> > Yes, I just updated it now.
> 
> Can you add a note about it in the Updating section in the Readme?

Done.

**Review by jaxesn — APPROVED** [review]
Awesome!

**Pull request merged**